### DISCUSSION TITLE
raw.githubusercontent.com refused a connection

### DISCUSSION THREAD

**Discussion by Bully85** [community/community#75179] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello i have a problem. i can´t open any raw.githubusercontent.com and can´t see image filse in the repo´s. only with 1 PC with the other pc in my network all works fin. I don´t find the problem can someone help me? my System is win11

**Comment by bleewn** [comment]
Response #2

**Comment by bleewn** [comment]
reponse #5

**Reply by Bully85** [reply]
To what extent should this help to solve my problem?

**Comment by Sintayew4** [comment]
Respons#75179

**Comment by bleewn** [comment]
Respons#75179

**Comment by Bully85** [comment]
ok it is sloved. it was in the hosts file blocked
